packages/application-generic/src/usecases/compile-email-template/compile-email-template.usecase.ts

```diff
@@ -149,7 +149,7 @@ export class CompileEmailTemplate {
   public static addPreheader(content: string): string {
     // "&nbsp;&zwnj;&nbsp;&zwnj;" is needed to spacing away the rest of the email from the preheader area in email clients
     return content?.replace(
-      /<body[^>]*>/g,
+      /<body\b[^>]*>/,
```

Comment:
```suggestion
      /<body\b[^\<\>]*?>/,
```
This one seems to be safe and I did some tests and seems to work. Let's try with this one.